docs/c_api.md

```diff
@@ -1658,7 +1658,13 @@ In WasmEdge, developers can create the `Function`, `Memory`, `Table`, and `Globa
     /* Register the WASI and WasmEdge_Process into the VM context. */
     WasmEdge_VMRegisterModuleFromImport(VMCxt, WasiObj);
     WasmEdge_VMRegisterModuleFromImport(VMCxt, ProcObj);
+    /* Get the WASI exit code. */
+    uint32_t ExitCode = WasmEdge_ImportObjectWASIGetExitCode(WasiObj);
+    /* The `ExitCode` will be `EXIT_SUCCESS`. */
```

Comment:
The `ExitCode` will be `EXIT_SUCCESS` if the execution has no error. Otherwise, it will return with the related exit code.